serverless/src/forwarder.ts

```diff
@@ -8,8 +8,9 @@ const LAMBDA_LOG_GROUP_PREFIX = "/aws/lambda/";
 const FN_SUB = "Fn::Sub";
 const FN_JOIN = "Fn::Join";
 const REF = "Ref";
-export const SUBSCRIPTION_FILTER_NAME = "datadog-serverless-macro-filter";
+const maxAllowableLogGroupSubscriptions = 2;
 
```

Comment:
Nit: What's this blank line doing here? I would move it below all of the constants.

Reply:
Also, looks like the convention in this file/project is for constants like this to be in `ALL_CAPS`

Reply:
Do you mean the blank line on line 12?

Reply:
I got rid of the space on line 12 but added one on line 13 to separate the interface definitions from the interface definitions as that was the convention when I cloned the repo.